### ISSUE TITLE
feat: Add tooltips to dashboard filter icons

### ISSUE THREAD

**Issue by ziuus** [R3ACTR/EventFlow#216]
The dashboard filter icons (Date, Status, etc.) lack descriptive tooltips, which can be confusing for new users and is suboptimal for accessibility. Adding simple tooltips on hover would improve the UX. We would like to work on this as part of OSQ.
